src/libecalc/presentation/exporter/queries.py

```diff
@@ -276,11 +283,15 @@ def query(
                             fuel_consumer_result: GeneratorSetResult = installation_graph.get_energy_result(
                                 fuel_consumer.id
                             )
+                            power_result = fuel_consumer_result.power
+
+                            if self.power_from_shore_output == PowerFromShoreOutputType.POWER_SUPPLY_ONSHORE:
```

Comment:
I assume this is intended, but it is not clear how this works, ie. the same query is used for 3 different scenarios, depending on what is being asked for?

Reply:
Yes, instead of having a new query for each extra PfS-column (all three columns are different power-outputs)